### PR TITLE
InterpreterOps: Extend SSAData size to accomodate 256-bit operations

### PR DESCRIPTION
**Pull request by lioncash**
Extends how the SSAData block is allocated so that 256-bit operations can be implemented safely.

One other point to bring up is the preferred path forward to supporting those operations in the interpreter. We use `__uint128_t` fairly extensively, but there is no `__uint256_t` equivalent to translate over to.  Currently I was thinking of using an existing solution like [this project](https://github.com/calccrypto/uint256_t) that provides a uint256_t type, but wanted to hear other ideas if anyone had them.

### CODE REVIEW

**Review by Sonicadvance1 — APPROVED** [review]
I'm fine with this approach, or just defining a struct type that is 256-bit wide so we can still index the same way. Doesn't really matter to me, everything still needs to get type cast regardless.
```cpp
struct Vector256Bit {
  __uint128_t Lower;
  __uint128_t Upper;
};
```

**Comment by Sonicadvance1** [comment]
Let me know if you want to keep it the current way and merge as-is, or if you want to switch it to some struct/union.

**Comment by lioncash** [comment]
Oh sorry! I meant to get back to this. I'll convert it over to a struct when I wake up so the indexing is simpler

File: External/FEXCore/Source/Interface/Core/Interpreter/InterpreterOps.cpp [hunk 1]
```diff
@@ -336,23 +336,28 @@ void InterpreterOps::Op_NoOp(FEXCore::IR::IROp_Header *IROp, IROpData *Data, IR:
...
-  OpData.State = Frame->Thread;
-  OpData.SSAData = alloca(ListSize * 16);
-  OpData.CurrentEntry = Frame->State.rip;
-  OpData.CurrentIR = CurrentIR;
-  OpData.StackEntry = StackEntry;
-  OpData.BlockIterator = CurrentIR->GetBlocks().begin();
-
-  // Clear them all to zero. Required for Zero-extend semantics
-  memset(OpData.SSAData, 0, ListSize * 16);
+  constexpr size_t ListEntrySizeInBytes = 32;
```
Comment (Sonicadvance1) [inline External/FEXCore/Source/Interface/Core/Interpreter/InterpreterOps.cpp:346]: Probably want this 32 to be `sizeof(InterpVector256)` now

Comment (lioncash) [reply External/FEXCore/Source/Interface/Core/Interpreter/InterpreterOps.cpp]: Done!

**Review by Sonicadvance1 — APPROVED** [review]

**Pull request merged**